### DISCUSSION TITLE
Suggestions generated from a "contains" rather than a "starts-with"

### DISCUSSION THREAD

**Discussion by cathalmsg** [digitalfortress-tech/typeahead-standalone#33] [Q&A] [answered]
Hi is there any way to have the filter on suggestions generated do so from a "contains" rather than a "starts-with"?

**Comment by niketpathak** [comment, marked as answer]
Hello, 

Generally, typeahead is able to filter suggestions with multiple words, for example, in the movies example on the [playground](https://typeahead.digitalfortress.tech/#playground?id=movies-example), if you search for _steel_, then the suggestion **Man of Steel** should rightly pop-up.

However, if you wish that suggestions should be filtered by any letter in a word, it is currently not possible due to the way the `Trie` data structure works. This is called fuzzy matching and there exists another [package](https://www.npmjs.com/package/fuzzysearch) that might suit your needs.
